app/views/admin/financial_reports/index.html.erb

```diff
@@ -1,39 +1,74 @@
-<% page_title @organisation.name + " financial reports" %>
-<div class="organisation-header">
-  <h1><%= @organisation.name %></h1>
-  <%= view_on_website_link_for @organisation %>
+<% content_for :page_title, "#{@organisation.name} financial reports" %>
+<% content_for :title, @organisation.name %>
+<% content_for :context, "Organisation" %>
+<% content_for :title_margin_bottom, 4 %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <p class="govuk-body"><%= view_on_website_link_for @organisation, class: "govuk-link" %></p>
+  </div>
 </div>
 
-<%= tab_navigation_for(@organisation) do %>
-  <h2 class="add-bottom-margin">Financial Reports for <%= @organisation.name %></h2>
-  <%= link_to "Add report", {action: 'new'}, {class: 'btn btn-default add-bottom-margin'} %>
-  <% if @organisation.financial_reports.present? %>
-    <table class="table table-bordered table-striped">
-      <thead>
-        <tr class="table-header">
-          <th>Year</th>
-          <th>Funding</th>
-          <th>Spending</th>
-          <th></th>
-          <th></th>
-        </tr>
-      </thead>
-      <tbody>
-        <% @organisation.financial_reports.each do |report| %>
-          <tr>
-            <td><%= report.year %></td>
-            <td><%= number_to_currency(report.funding, precision: 0, unit: '&pound;'.html_safe) %></td>
-            <td><%= number_to_currency(report.spending, precision: 0, unit: '&pound;'.html_safe) %></td>
-            <td><%= link_to 'Edit', edit_admin_organisation_financial_report_path(@organisation, report), method: :get %></td>
-            <td><%= link_to 'Destroy',
-                      admin_organisation_financial_report_path(@organisation, report),
-                      data: { confirm: "Are you sure?" },
-                      method: :delete %></td>
-          </tr>
-        <% end %>
-      </tbody>
-    </table>
-  <% else %>
-    <p class="no-content no-content-bordered">No financial reports</p>
-  <% end %>
-<% end %>
+<%= render "components/secondary_navigation", {
+  aria_label: "Organisation navigation tabs",
+  items: secondary_navigation_tabs_items(@organisation, request.path)
+} %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= render "govuk_publishing_components/components/heading", {
+      text: "Financial Reports",
+      margin_bottom: 6
+    } %>
+
+    <%= render "govuk_publishing_components/components/button", {
+      text: "Create new report",
+      href: {action: 'new'},
```

Comment:
i can see this works but i wonder if we can use

```
new_admin_organisation_financial_report_path(@organisation)
```

We already use a few patterns for creating href's so we can use one of those

Reply:
Also Ruth has asked us to track all primary buttons so can we add 

```
data_attributes: {
        module: "gem-track-click",
        "track-category": "form-button",
        "track-action": "create-financial-report-button",
        "track-label": "Create new report"
      }
```